EventView-1/src/main/java/com/eventview/model/Users.java

```diff
@@ -40,28 +40,28 @@ public Users() {
 
     }
 
-    public Integer getUserid() {
-        return userid;
+    public Integer getUserId() {
+        return userId;
     }
 
-    public void setUserid(Integer userid) {
-        this.userid = userid;
+    public void setUserId(Integer userid) {
+        this.userId = userid;
     }
 
-    public String getFname() {
-        return fname;
+    public String getFName() {
+        return fName;
     }
```

Comment:
how come this is working for you?

Reply:
yeah it is working this way.
what happened is when i generated getter setter methods it gave methods "getfName" and "getlName" which i then changed to "getFName" and "getLName" for easy of reading